File: BaseWork/app/src/main/java/com/zihuan/app/UserManager.java
package com.zihuan.app;


import com.zihuan.app.model.UserEntity;
import com.zihuan.app.u.U;

public class UserManager {

    public UserEntity userData = new UserEntity();

    private UserManager() {
    }

    private static UserManager mUserManager;

    public static UserManager getInstance() {
        if (mUserManager == null) {
            mUserManager = new UserManager();
        }
        return mUserManager;
    }

    public void save(UserEntity user) {
        userData = user;
        U.savePreferences("uid",userData.getUid());
        U.savePreferences("token",userData.getToken());
    }


}

File: BaseWork/app/src/main/java/com/zihuan/app/UserManager.kt
package com.zihuan.app


import com.zihuan.app.model.UserEntity
import com.zihuan.utils.cmhlibrary.savePreference

object UserManager {

    var userData = UserEntity()

    fun save(user: UserEntity) {
        userData = user
        userData.uid?.savePreference("uid")
        userData.token?.savePreference("token")
    }


}
